lib/MusicBrainz/Server/Edit/CheckForConflicts.pm

```diff
@@ -123,7 +123,7 @@ sub merge_changes {
                 );
 
             # Stores a mapping of the JSON serialization to the object value
-            my %hashed = map { @$_ } ($new, $ancestor, $current);
```

Comment:
So...ideally the same hash key should map to the exact same value, and these would be equivalent. But that wasn't the case here, and I have to wonder if this resulted in any other bugs.